documentation/Functions.md

```diff
@@ -312,13 +312,26 @@ If the tutorial should be tested on the console environment, you have to specify
 ***
 
 ### downloadFile <a name="downloadFile"></a>
+This function is used to download a file from an external URL which is passed as a parameter.
```

Comment:
Remove `which is passed as a parameter`, you got it twice, it will be explained in the parameter part.

Reply:
done